src/core_plugins/kibana/public/visualize/wizard/wizard.js

```diff
@@ -159,9 +159,13 @@ module.controller('VisualizeWizardStep1', function ($scope, $route, kbnUrl, Priv
     //labs is more important in this regard.
     let prefix = '';
     if (type.stage === 'lab') {
-      prefix = '(Lab)';
+      prefix = i18n('kbn.visualize.visualizationWizard.labTooltipPrefixText', {
+        defaultMessage: '(Lab)'
+      });
     } else if (type.stage === 'experimental') {
-      prefix = '(Experimental)';
+      prefix = i18n('kbn.visualize.visualizationWizard.experimentalTooltipPrefixText', {
```

Comment:
maybe it is better to pass type.description as param via values? And it would be experimentalDescription